apps/scan/frontend/src/screens/cast_vote_record_sync_required_screen.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
```

Comment:
This file began with a copy-paste of the prior `cast_vote_record_sync_modal.tsx` file, followed by some edits to 1) make the UI full screen and 2) move some state management to the parent.

Reply:
What happened to the prior modal file? I don't see it deleted in this PR

Reply:
🙈 Oops! https://github.com/votingworks/vxsuite/pull/4279/commits/06da19102c97e59396685e8674c67a432460f497